src/Eccube/Service/SystemService.php

```diff
@@ -68,4 +68,16 @@ public function getDbversion()
 
         return $prefix.$version;
     }
+
+    /**
+     * @return null|string
+     */
+    public function composerVersion(){
+        $composer = exec('composer -V');
```

Comment:
1. please write in composer service (not in system service)
2. use `composer.phar`